packages/extension-browser/README.md

```diff
@@ -0,0 +1,4 @@
+# webhint browser extension (`@hint/extension-browser`)
+
+A web extension and connector to analyze a page running in a browser
+with `webhint`.
```

Comment:
Maybe add a new issue to have more documentation? We should do some work in the website as well to get this data.

Reply:
Filed as #1508.